GitUtils.ps1

```diff
@@ -1,11 +1,79 @@
 # Inspired by Mark Embling
 # http://www.markembling.info/view/my-ideal-powershell-prompt-with-git-integration
 
+<#
+.SYNOPSIS
+    Test if directory is within Git working tree.
+
+.DESCRIPTION
+    Returns True if the directory is in a Git working tree.
+
+    This can be used in a pipeline to check if a specific 
+    location is a part of a Git working tree. 
+#>
+function Test-GitWorkingTree {
+    [CmdletBinding()]
+    param(
+        [Parameter(ValueFromPipeline=$true,Position=0)]
+        [AllowEmptyCollection()]
+        [String[]] $Path,
+
+        [Parameter()]
+        [String] $GitDir = $Env:GIT_DIR
+    )
+    PROCESS {
+        $p = ?? $Path @(Get-Location)

```

Comment:
True again. It must have been an artifact of some experiments I tried... probably safe to delete...